projects/btldr/ecus.bzl

```diff
@@ -11,6 +13,9 @@ ECUS = {
     "brakelight_bspd": {
         "btldr_id": "0x718",
     },
+    "brakes": {
```

Comment:
What's the distinction between `brakelight_bspd` and `brakes`?

Reply:
brakelight_bspd is the name from mkv, brakes is the name from mkvi. It's a thing with how the btldr finds it's ID using the name of the cc_firmware build target. It's not the best way to go about it but it worked and I don't think I'm going to spend much time finding a better solution for now haha.

Reply:
fair enough